tools/wpt/browser.py

```diff
@@ -431,8 +431,45 @@ class Servo(Browser):
     product = "servo"
     requirements = "requirements_servo.txt"
 
+    def platform_string(self):
+        platform = {
+            "Linux": "linux",
+            "Windows": "win",
+            "Darwin": "mac"
+        }.get(uname[0])
+
+        if platform is None:
+            raise ValueError("Unable to construct a valid Servo package for current platform")
+
+        return "%s" % (platform)
+
+    def find_extension(self, platform):
+        if platform == "linux":
+            extension = ".tar.gz"
+        elif platform == "win":
+            extension = ".msi"
+        else:
+            extension = ".dmg"
+
+        return "%s" % (extension)
+
     def install(self, dest=None):
-        raise NotImplementedError
+        """Install latest Browser Engine."""
+        if dest is None:
+            dest = os.pwd
+
+        extension = self.find_extension(self.platform_string())
+        url = "https://download.servo.org/nightly/%s/servo-latest%s" % (self.platform_string(), extension)
+
+        if self.platform_string() == "linux":
```

Comment:
Since we only support linux right now, we can remove any code that checks for the platform or tries to support other platforms.

Reply:
And if we keep this, maybe `platform_components` or whatever should return the `unzip`/`untar` function as well, to localize the dependency on the kind of files that are used on each platform.

Reply:
I am not sure about returning `unzip / untar` function from `platform_components` since `platform` and `extension` values are used to form a url which downloads the required file that is extracted using `untar / unzip` function. Would not including url or filename in `platform_components` make the function more complicated?

Reply:
@kriti21 The previous comment meant something like this:
```python
def platform_components(self):
    if ...:
        return ("linux", ".tar.gz", untar)

platform, extension, decompress = self.platform_components()
url = "..." % (platform, extension)
decompress(get(url).raw, dest=dest)
```
Does that make sense?

Reply:
Oh okay! Thanks :)